### PR TITLE
Stats: Tables: Roles/Operators: CSV generation

### PR DESCRIPTION
**Pull request by kamsi**
Server-side generated CSV representation of view implemented at #1770 

It should generate sheet of data representing same filtered version of view as one from which CSV request was initiated (query parameters needs to be passed to CSV url).

Still the page at which view is at is irrelevant, in all cases we should generate rows for whole selected period.

CSV representation should follow description in that comment: https://github.com/egovernment/eregistrations/issues/1759#issuecomment-275636701

### CODE REVIEW

File: server/csv-renderers/statistics-flow-operators.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  , _                = require('mano').i18n.bind('Statistics operators csv')
+  , usersFullNameMap = require('../lib/user-full-names-map');
+
+module.exports = function (result, config) {
+	ensureObject(config);
+	var data, mode;
+	mode = modes.get(config.mode || 'daily');
+	data = [
+		[
+			mode.labelNoun,
```
Comment (medikoo) [inline server/csv-renderers/statistics-flow-operators.js]: This doesn't reflect expected format, as described in this comment -> https://github.com/egovernment/eregistrations/issues/1759#issuecomment-275636701

First column was supposed to state step, and then we should list data for all steps, one after another

**Comment by medikoo** [comment]
@mtuchowski I'm assigning it to you, as @kamsi is on long vacations

**Comment by mtuchowski** [comment]
Ready to merge to grouping PR https://github.com/egovernment/eregistrations/pull/1793 after https://github.com/egovernment/eregistrations/pull/1769 is done.

File: server/csv-renderers/statistics-flow-operators.js (hunk 1 above)
Comment (mtuchowski) [inline server/csv-renderers/statistics-flow-operators.js]: As per talk with @roxanadina this no longer applies: The CVS export should contain exactly the data shown on the page (same as PDF).

**Pull request merged**